### PR TITLE
612 faq page css update

### PR DESCRIPTION
**Pull request by Tokikko**
#612

### CODE REVIEW

File: design/scss/_variables.scss [hunk 1]
```diff
@@ -34,7 +34,11 @@ $danger: #ff5c6f !default;
 $light: #e9ecef !default;
 $dark: #313131 !default;
 $gray: #6c757d !default;
-
+// Override bootstrap acordion arrow color
+$accordion-icon-active-color: #FFF;
+$accordion-icon-color: #FFF;
+// Override bootstrap acordion arrow color END
+$acordion-collapsed-color: rgb(52 78 103);
```
Comment (nick-w-nick) [inline design/scss/_variables.scss:41]:
```suggestion
$accordion-collapsed-color: rgb(52 78 103);
```

minor spelling fix

File: design/scss/custom/_faq.scss [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+
+    .accordion-button:focus{
+        border-color: inherit;
+        box-shadow: none;
+    }
+    
 	.collapsed {
 		color: $white;
-		background-color: $primary;
+		background-color: $acordion-collapsed-color;
```
Comment (nick-w-nick) [inline design/scss/custom/_faq.scss:18]:
```suggestion
		background-color: $accordion-collapsed-color;
```

minor spelling fix to go along with [this suggestion](https://github.com/donategifts/donategifts/pull/651/files#r1300673075)

File: design/scss/_variables.scss (hunk 1 above)
Comment (stacysealky) [reply design/scss/_variables.scss]: nice catch @nick-w-nick thank you

**Review by stacysealky — APPROVED** [review]
Looks good, will merge

**Pull request merged**